opm/autodiff/WellInterface_impl.hpp

```diff
@@ -1171,8 +1327,9 @@ namespace Opm
     template<typename TypeTag>
     bool
     WellInterface<TypeTag>::
-    solveWellEqUntilConverged(Simulator& ebosSimulator,
+    solveWellEqUntilConverged(const Simulator& ebosSimulator,
                               const std::vector<double>& B_avg,
```

Comment:
Why was the simulator non-const here and several other places in the code?

Reply:
I think it is because the function `assembleWellEq` might change the Jacobian, which is part of `ebosSimulator`? I did not re-read the code, excuse me it is wrong.

Reply:
That is no-longer the case since the Jacobian part is a separate step. So then it should be changed to const.

Reply:
okay. If it compiles with `const` there, then `const` should be there.